src/Riok.Mapperly/Diagnostics/DiagnosticDescriptors.cs

```diff
@@ -293,4 +293,31 @@ internal static class DiagnosticDescriptors
         DiagnosticSeverity.Info,
         true
     );
+
+    public static readonly DiagnosticDescriptor DerivedSourceTypeDuplicated = new DiagnosticDescriptor(
+        "RMG034",
+        "Derived source type is specified multiple times, a source type may only be specified once",
+        "Derived source type {0} is duplicated",
```

Comment:
I think this message should also be changed

Reply:
Fixed.